src/content/tutorial/react-step-3/react-step-3.mdx

```diff
@@ -301,233 +266,188 @@ const getRowItems = rows =>
+The page will look the same as we're still rendering our static example rows, but if you view your browser's console (e.g. [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/)), you should see the response from GitHub!
 
-_Note: The Pagination component isn’t inherently connected in any way to the DataTable, we need to tell it what to do when a change occurs using the onChange prop. This includes both page size changes and displaying different rows._.
+## Populate data table
 
-## Create component
+Now that we have that data, let's populate the data table. Replace `console.log(organization);` with the following that destructures the `organization` object because we're only interested in `organization.repositories`. Once we have the respositories, we can call our `getRowItems()` helper to build the data table rows.
 
-Now we'll need to consume our fetched data. In React, we generally want to separate our data fetching logic from our presentational or visual content. We'll separate our visual content into the `RepoPage/RepoTable.js` file. It's important to keep in mind that because we only render our `RepoTable` when we have our data, we don't need to be as safe with `null`/`undefined` checking our individual row properties. We've established that they exist given the nature of our GraphQL query.
+##### src/content/RepoPage/RepoPage.js
 
-### Create file
+```javascript
+// If we're here, we've got our data!
+const { repositories } = organization;
+const rows = getRowItems(repositories.nodes);
+```
 
-Go ahead and create a new `RepoTable.js` file in `src/content/RepoPage`.
+Then, towards the top of `RepoPage.js` delete the `rows` array because we no longer need the example rows.
 
-### Imports
+We need to modify one apsect of the data table because if you expand a row, it says `Row description`. We want to update that with the repository description coming from the GitHub API. To do so, in `RepoTable.js`, add a helper function as the first lines inside the `RepoTable`. This should go immediately before the component's `return()`.
 
-Import the various components that we'll need to render our table at the top of `RepoTable.js`.
+##### src/content/RepoPage/RepoTable.js
 
 ```javascript
-import React from 'react';
-import {
-  DataTable,
-  TableContainer,
-  Table,
-  TableHead,
-  TableRow,
-  TableExpandHeader,
-  TableHeader,
-  TableBody,
-  TableExpandRow,
-  TableCell,
-  TableExpandedRow,
-} from 'carbon-components-react';
+const getRowDescription = rowId => {
+  const row = rows.find(({ id }) => id === rowId);
+  return row ? row.description : '';
+};
 ```
 
-### Component cleanup
+Finally, in `RepoTable.js`, replace `<p>Row description</p>` with:
 
-The data table component and its pieces use a common React pattern called [render props](https://reactjs.org/docs/render-props.html). This a really powerful way for libraries to give developers control of rendering and manipulating their data.
```

Comment:
This is critical information and the biggest source of issues in slack for DataTable